react-components/toolbar-filter/assets/table.js

```diff
@@ -94,4 +94,4 @@ class AppTable extends React.Component {
   }
 }
 
-export default AppTable;
+export default DemoTable;
```

Comment:
```suggestion
export default `DemoTable`;
```

Reply:
same comment here.

Reply:
The ticks can only be added to files that have a .md extension

Reply:
got it..thanks